test/sequential/test-fs-readfile.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+// In case openssl is not available
+function slowRandomBytes(len) {
+  const buf = Buffer.alloc(len);
+  for (let i = 0; i < len; i++)
+    buf[i] = Math.random();
+  return buf;
+}
+
+common.refreshTmpDir();
+
+const fileInfo = [ { 'name': path.join(common.tmpDir, '1Kfile.txt'),
+                     'len': 1024,
+},
+                   { 'name': path.join(common.tmpDir, '64Kfile.txt'),
+                     'len': 64 * 1024,
+                   },
+                   { 'name': path.join(common.tmpDir, '1Mfile.txt'),
+                     'len': 1 * 1024 * 1024,
+                   },
+];
+
+/* Populate each fileInfo (and file) with unique fill. */
+for (let i = 0; i < fileInfo.length; i++) {
+  fileInfo[i].fill = slowRandomBytes(fileInfo[i].len);
```

Comment:
I'd just `fileInfo[i].fill = Buffer.allocUnsafe(fileInfo[i].len).fill(i);`

As well, you can DRY these loops like this, which is also a little more idiomatic:
```js
for (const e of fileInfo) {
  e.fill = /* ... */;
  fs.writeFileSync(e.name, e.fill);
}
```

Reply:
Great, thanks.